src/components/Button.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+const Button = styled.button`
```

Comment:
Can you find common props with AddGameButton and extend or extend this in AddGameButton ?

Reply:
You are right that it's the same look as the other buttons, so we will try to find a common ground for this one. The problem is simply that this particular button is composed of two parts so it is a bit more complicated than the other ones. 
ALSO this button will change into a "remove button" too, so it would be re-used anyway.